hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Tables/ReviewOrgUnitChangesTable.tsx

```diff
@@ -118,12 +123,16 @@ const useColumns = (): Column[] => {
                 accessor: 'created_by',
                 Cell: UserCell,
             },
-            {
-                Header: formatMessage(MESSAGES.paymentStatus),
-                id: 'payment_status',
-                accessor: 'payment_status',
-                Cell: BreakWordCell,
-            },
+            ...(modules.includes(PAYMENTS_MODULE)
+                ? [
+                      {
+                          Header: formatMessage(MESSAGES.paymentStatus),
+                          id: 'payment_status',
+                          accessor: 'payment_status',
+                          Cell: BreakWordCell,
+                      },
+                  ]
+                : []),
```

Comment:
this is working but i would prefer adding a slice to the table instead of this, it makes the code a bit unclear